### ISSUE TITLE
Request: Add Euro currency

### ISSUE THREAD

**Issue by vtaxiarchis** [lionsharecapital/lionshare-desktop#103]
It would be very convenient if it was able to support more currencies, such as EURO.

**Comment by trych** [comment]
That would be great, indeed.

**Comment by ispyhumanfly** [comment]
I agree, I'd also like it to handle the top world currencies, as well as aid in managing other holdings.
